ResearchUXFactory/NSDate+ISO8601Format.m

```diff
@@ -72,6 +72,34 @@ + (NSDateFormatter *)ISO8601DateOnlyformatter
     return formatter;
 }
 
++ (NSDateFormatter *)UnitedStatesDateOnlyformatter
```

Comment:
Because this is based on the locale of the researcher who is adding this into the UI. Basically, it's a hack-around for everything being converted to a string by the Web Researcher UI. I felt that it was "okay" to do this while our researchers are all Americans entering US-locale strings but I didn't want to assume further than that.